tests/frontier/opcodes/test_call_code_gas.py

```diff
@@ -96,26 +108,24 @@ def pre(callee_code, caller_code) -> Dict[str, Account]:  # noqa: D103
 
 
 @pytest.fixture
-def post(caller_gas) -> Dict[str, Account]:  # noqa: D103
+def post(sufficient_gas: bool) -> Dict[str, Account]:  # noqa: D103
     return {
-        to_address(Accounts.caller): Account(
-            storage={0x00: (0x01 if caller_gas >= 0x3D50 else 0x00)}
-        ),
+        to_address(Accounts.caller): Account(storage={0x00: 0x01 if sufficient_gas else 0x00}),
     }
 
 
 @pytest.mark.parametrize(
-    "caller_gas",
-    [0x2D5F, 0x3D50],
-    ids=["insufficient", "sufficient"],
-)
-@pytest.mark.parametrize(
-    "caller_type",
-    [Op.CALL, Op.CALLCODE],
+    "caller_type, caller_gas, sufficient_gas",
```

Comment:
Could we rename these params?
```suggestion
    "call_opcode, caller_gas, use_sufficient_gas",
```
Maybe `opcode` is sufficient?

Excuse pedanticness.

Reply:
Sounds good - thanks!

Reply:
Using `caller_opcode` to specify its the opcode use do call the caller contract